src/app/pages/resources/dashboard/dashboard.jsx

```diff
@@ -90,9 +120,13 @@ function ChplDashboard() {
                           key={`${report.reportKey}-button`}
                           style={{ justifyContent: 'flex-start' }}
                           color="primary"
+                          className={`${classes.menuButton} ${activeReportKey === report.reportKey ? 'active' : ''}`}
                           onClick={() => handleReportChange(report.reportKey)}
+                          id={`report-${report.reportKey}`}
+                          fullWidth
+                          variant="text"
                         >
-                          { report.title }
+                          {report.title}
```

Comment:
```suggestion
                          { report.title }
```

Unless it's required to not have the space (I think classNames and some HTML elements are like that) it's slightly easier to read with some extra space